config/utils_variables.py

```diff
@@ -146,4 +146,19 @@
 
 #workflow tables
 workflow = 'workflows'
-workflowstages = 'workflow_stages'
\ No newline at end of file
+workflowstages = 'workflow_stages'
+
+#Define default work flow name
+default_workflow_name = 'sales'
+
+# Define the stages for the default workflow
+default_workflow_stages = {
+    1: 'sale_order',
+    2: 'dispatch',
+    3: 'invoice'
+}
+
+# Accessing values
+first_stage = default_workflow_stages[1]  # 'sale_order'
+second_stage = default_workflow_stages[2]  # 'dispatch'
+third_stage = default_workflow_stages[3]  # 'invoice'
```

Comment:
These are not needed, you can access them directly oin the code. right?

Reply:
yeah yes Thulasi.